### PR TITLE
SUEGGSWIF COMMAND!!

### PR DESCRIPTION
**Pull request by OviRei**
EEEEEEEEEEEEEEEE

### CODE REVIEW

File: source/Commands/SuggestCommand.ts [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    public commandOptions: AbstractCommandOptions = new SuggestCommandOptions();
+    public async runInternal(bot: Client, message: Message, messageArray: Array<string>)
+    {  
+        const args = message.content.slice(AbstractCommandOptions.prefix.length).split(/ +/);
+        const botmessage: string = args.join(' ');
+        let suggestEmbed: MessageEmbed = new MessageEmbed()
+        .setTitle(`${message.author.tag}`)
+        .setAuthor('User Suggestion: ')
+        .setDescription(`${botmessage}`)
+        .setTimestamp(new Date)
```
Comment (Montori) [inline source/Commands/SuggestCommand.ts]: `new Date()`

File: source/Commands/SuggestCommand.ts [hunk 2]
```diff
@@ -0,0 +1,38 @@
+import {AbstractCommand} from "./AbstractCommand";
+import {Client, Message, MessageEmbed} from 'discord.js';
+import { AbstractCommandOptions } from "../Material/AbstractCommandOptions";
+
+export class SuggestCommand extends AbstractCommand
+{
+    public commandOptions: AbstractCommandOptions = new SuggestCommandOptions();
+    public async runInternal(bot: Client, message: Message, messageArray: Array<string>)
+    {  
+        const args = message.content.slice(AbstractCommandOptions.prefix.length).split(/ +/);
```
Comment (Montori) [inline source/Commands/SuggestCommand.ts]: redundant just use `messageArray`

File: source/Commands/SuggestCommand.ts [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+class SuggestCommandOptions extends AbstractCommandOptions{
+    public commandName: string;
+    public description: string;
+    public usage: string;
+
+    constructor()
+    {
+        super();
+        this.commandName = "suggest";
+        this.description = "its a suggest cmd...";
```
Comment (Montori) [inline source/Commands/SuggestCommand.ts]: add a real description

File: source/Commands/SuggestCommand.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+        this.usage = `${AbstractCommandOptions.prefix}suggest`;
```
Comment (Montori) [inline source/Commands/SuggestCommand.ts]: add the real usage `${AbstractCommandOptions.prefix}suggest {suggestion ...}`

File: source/Commands/SuggestCommand.ts [hunk 5]
```diff
@@ -0,0 +1,38 @@
...
+    }
```
Comment (Montori) [inline source/Commands/SuggestCommand.ts:38]: add a cooldown with `this.cooldown = 1800;`

**Review by Montori — CHANGES_REQUESTED** [review]

**Comment by OviRei** [comment]
Changed have been made to the code.

**Pull request merged**